docs/layers/github-actions/design-decisions/decide-on-self-hosted-runner-architecture.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Decide on Self-Hosted Runner Architecture"
+sidebar_label: Runner Architecture
+description: Decide how to create self-hosted runners
+---
+
+import Intro from "@site/src/components/Intro";
+import Note from '@site/src/components/Note';
+import Steps from "@site/src/components/Steps";
+import Step from "@site/src/components/Step";
+import StepNumber from "@site/src/components/StepNumber";
+
+<Intro>
+  Self-hosted runners are custom runners that we use to run GitHub Actions workflows. We can create these runners a
+  number of ways.
```

Comment:
```suggestion
Decide on how to operate self-hosted runners that are used to run GitHub Actions workflows. These runners can be set up in various ways and allow us to avoid platform fees while running CI jobs in private infrastructure, enabling access to VPC resources. This approach is ideal for private repositories, providing control over instance size, architecture, and control costs by leveraging spot instances. The right choice depends on your platform, whether you’re using predominantly EKS, ECS, or Lambda.”
```